Makefile

```diff
@@ -304,6 +312,16 @@ load-on-multi-cluster: ## Load the docker image into both kind clusters.
 	KIND=$(KIND) bash -c 'export KIND_CLUSTER_NAME=pe-kind-a && source clab/common.sh && load_local_image_to_kind ${IMG} router-a'
 	KIND=$(KIND) bash -c 'export KIND_CLUSTER_NAME=pe-kind-b && source clab/common.sh && load_local_image_to_kind ${IMG} router-b'
 
+##@ Kind Node Image
+
+.PHONY: kind-node-image-build
+kind-node-image-build: ## Build custom kind node image with OVS
```

Comment:
Let's always build before deploy. As @qinqon said in the other PR, the caching mechanism should keep this quick in case we don't need to rebuild

Reply:
Either that as a dependency of `clab-cluster` and `clab-multicluster` or we maintain and publish the image.
If we publish the image, I'd do that as part of CI without allowing users to push